src/annotator/notebook.js

```diff
@@ -2,17 +2,20 @@ import { createShadowRoot } from './util/shadow-root';
 import { render } from 'preact';
 import NotebookModal from './components/NotebookModal';
 
-/** @typedef {import('../types/annotator').Destroyable} Destroyable */
+/**
+ * @typedef {import('../types/annotator').Destroyable} Destroyable
+ * @typedef {import('./components/NotebookModal').NotebookConfig} NotebookConfig
+ */
 
 /** @implements {Destroyable} */
 export class Notebook {
   /**
    * @param {HTMLElement} element
    * @param {import('./util/emitter').EventBus} eventBus -
    *   Enables communication between components sharing the same eventBus
-   * @param {Record<string, any>} config
+   * @param {NotebookConfig} config
    */
-  constructor(element, eventBus, config = {}) {
+  constructor(element, eventBus, config) {
```

Comment:
The default argument (`{}`) was removed here because it was not used and is missing the required `notebookAppUrl` property.